specification/databox/resource-manager/Microsoft.DataBox/stable/2020-10-01/databox.json

```diff
@@ -0,0 +1,4772 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-04-01",
```

Comment:
version is 2020-04-01 and why are we checking into  specification/databox/resource-manager/Microsoft.DataBox/stable/2020-10-01/databox.json ? we should checkin to  specification/databox/resource-manager/Microsoft.DataBox/stable/2020-04-01/databox.json

Reply:
Corrected now, Thank you